client/components/marketing-page-feature/index.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode, FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	children: ReactNode;
+	description: string;
+	disclaimer?: string;
+	imageAlt?: string;
+	imagePath?: string;
+	title: string;
+}
```

Comment:
we've done a few of these now. do you think there's merit in calling these `ExternalProps` since they are the ones expected to be passed in from the parent component vs. from the `connect()` wrapper?

`Props` is terse and easy, `ExternalProps` is longer but maybe clearer…

Reply:
I am not sure, my mental model would assume that the `ExternalProps` are coming from Redux or some other source and the `Props` being the ones it gets passed from the parent. Just my 2 cents.

Reply:
Okay. I think I've seen that terminology in a variety of React+TypeScript projects. I'll investigate further to see what the industry is using.